src/Microsoft.AspNetCore.Identity/IdentityOptions.cs

```diff
@@ -11,6 +13,18 @@ namespace Microsoft.AspNetCore.Builder
     /// </summary>
     public class IdentityOptions
     {
+        public IdentityOptions(IEnumerable<IConfigureOptions<IdentityOptions>> configureOptions = null)
```

Comment:
@HaoK I realize an Annoucement on this is forthcoming, but can you commnent on what the delta will be for the current pattern ...
```c#
services.AddOptions();
services.Configure<AppOptions>(appOptions =>
{
    appOptions.Key = "value";
    ...
});
```